runners/flip

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
This file needs to be marked as executable (`chmod +x runners/flip`)

Reply:
Well, GitHub doesn't let you upload executable files (whenever I try to use git to commit, all the executable information is lost, so marking `chmod +x` doesn't really help). Can you explain how you did it?

Reply:
Finally knew how to do it. https://www.augmentedmind.de/2020/05/10/commit-executable-shell-scripts-with-git/